### DISCUSSION TITLE
How can I filter and add tags for logs coming from http input?

### DISCUSSION THREAD

**Discussion by ylbeethoven** [fluent/fluent-bit#9017] [Q&A]
Hi everyone,

I have a use case that I have two application sending JSON logs to fluent-bit via http input.

I want to differentiate logs from these two applications by `user-agent` header and then appley different tags so I can use different outputs.

I've tried rewrite_tags and even Lua but not much luck.

Does anyone know if this is achievable? Or should I use two http input to receive logs from different sources?

Thank you.

**Comment by patrick-stephens** [comment]
HTTP input supports the tags coming from the sender: https://docs.fluentbit.io/manual/pipeline/inputs/http#how-to-set-tag
You can also specify a key to use. That seems the best way to do it to me.
